core-services/workflow/src/main/java/org/silverpeas/core/workflow/engine/WorkflowEngineTask.java

```diff
@@ -23,7 +23,10 @@
  */
 package org.silverpeas.core.workflow.engine;
 
-import org.silverpeas.core.cache.service.CacheServiceProvider;
+import org.silverpeas.core.SilverpeasRuntimeException;
+import org.silverpeas.core.thread.ManagedThreadPool;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.silverpeas.core.thread.ManagedThreadPool'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)